src/coprocessor/endpoint.rs

```diff
@@ -67,35 +68,55 @@ pub struct Host {
     sched: Scheduler<Task>,
     reqs: HashMap<u64, Vec<RequestTask>>,
     last_req_id: u64,
-    pool: ThreadPool<DummyContext>,
-    low_priority_pool: ThreadPool<DummyContext>,
-    high_priority_pool: ThreadPool<DummyContext>,
+    pool: ThreadPool<CopContext>,
+    low_priority_pool: ThreadPool<CopContext>,
+    high_priority_pool: ThreadPool<CopContext>,
     max_running_task_count: usize,
 }
 
-// TODO: remove these dummy structures.
 #[derive(Clone)]
-struct DummyContext {}
+struct CopContext {
+    raftstore_sender: Option<Sender<Statistics>>,
```

Comment:
raftstore only needs read statistics, but not the whole Statistics. 

Btw, how do we know the read statistics for different region?